commit a088d550a14074457c669d174a0237018a0bb4b1
Author: jim-p <jimp@netgate.com>
Date:   Mon Mar 4 15:48:56 2024 -0500

    Fix some SFP module info fields. Fixes #15112
    
    Text format changed slightly in ifconfig, so regex patterns had to be
    changed to match

diff --git a/src/etc/inc/pfsense-utils.inc b/src/etc/inc/pfsense-utils.inc
index 13feee5e54..7a1737e518 100644
--- a/src/etc/inc/pfsense-utils.inc
+++ b/src/etc/inc/pfsense-utils.inc
@@ -1825,11 +1825,11 @@ function get_interface_info($ifdescr) {
 			if (preg_match("/vendor: (.*)$/", $ici, $matches)) {
 				$ifinfo['vendor'] = $matches[1];
 			}
-			if (preg_match("/module temperature: (.*) Voltage: (.*)$/", $ici, $matches)) {
+			if (preg_match("/module temperature: (.*) voltage: (.*)$/", $ici, $matches)) {
 				$ifinfo['temperature'] = $matches[1];
 				$ifinfo['voltage'] = $matches[2];
 			}
-			if (preg_match("/RX: (.*) TX: (.*)$/", $ici, $matches)) {
+			if (preg_match("/RX power: (.*) TX bias: (.*)$/", $ici, $matches)) {
 				$ifinfo['rx'] = $matches[1];
 				$ifinfo['tx'] = $matches[2];
 			}
diff --git a/src/usr/local/www/status_interfaces.php b/src/usr/local/www/status_interfaces.php
index 67a0c95447..f67f2ad3ac 100644
--- a/src/usr/local/www/status_interfaces.php
+++ b/src/usr/local/www/status_interfaces.php
@@ -206,7 +206,7 @@ foreach ($ifdescrs as $ifdescr => $ifname):
 				showDef($ifinfo['temperature'], gettext("Temperature"), $ifinfo['temperature']);
 			}
 			if ($ifinfo['voltage']) {
-				showDef($ifinfo['voltage'], gettext("voltage"), $ifinfo['voltage']);
+				showDef($ifinfo['voltage'], gettext("Voltage"), $ifinfo['voltage']);
 			}
 			if ($ifinfo['rx']) {
 				showDef($ifinfo['rx'], gettext("RX"), $ifinfo['rx']);
